content/en/agent/kubernetes/tag.md

```diff
@@ -64,6 +64,15 @@ For example, you could set up:
 DD_KUBERNETES_NODE_LABELS_AS_TAGS='{"app":"kube_app"}'
 ```
 
+For Agent v7.24.0+, use the following environment variable configuration to add all node labels as tags to your metrics. In this example, the tags names are prefixed by `<PREFIX>_`:
+
+```shell
+DD_KUBERNETES_NODE_LABELS_AS_TAGS='{"*":"<PREFIX>_%%label%%"}'
+```
+
+**Note**: Using this method may increase the number of [custom metrics][1] for your organization and impact your billing.
```

Comment:
This option will also add node labels as tags for integration metrics (that are not custom metrics), correct? If so,  we should probably clarify that IF custom metrics are sent, it may increase their number and may have an impact on their billing

Reply:
correct, thanks for pointing out!
btw I copy/pasted the same note that we already have for pod labels as tags (same file), I can update the 3 notes - could you (or the docs team) please suggest what should be written there exactly?

Reply:
thanks - PR updated